tests/RedisConfigs/Sentinel/sentinel-26379.conf

```diff
@@ -1,5 +1,7 @@
 port 26379
+#requirepass "changeme"
```

Comment:
This seems to be a weird addition from testing perhaps? Let's revert all these commented config changes please - PR doesn't need to touch those files :)

Reply:
Same reason as above.